src/EnergyPlus/SetPointManager.cc

```diff
@@ -6130,6 +6142,20 @@ void DefineOutsideAirSetPointManager::calculate(EnergyPlusData &state)
         OutHighTemp = this->OutHigh1;
         SetTempAtOutLow = this->OutLowSetPt1;
         SetTempAtOutHigh = this->OutHighSetPt1;
+        if ((this->SchedPtr > 0) && (SchedVal != 1.0)) { // Since schedule is optional, only check this if the user entered a schedule
+            ++this->setPtErrorCount;
```

Comment:
A fine check here, and this will be hit even if the user manipulates the schedule at runtime with actuators.  :+1:

Reply:
I wonder if someone will complain that this code may never hit _exactly_ 1.0.

Reply:
Interesting. If the compiler interprets a 1 or a 2 double precision as 0.99999999999 or 1.99999999999 then I would think these checks would also see the value as 1 or 2?

Reply:
IKR? _Exactly_.  But if I put a tolerance in then we have a messy and time consuming check against tolerance.  Do we have a "best practices" for a situation like this that we've agreed upon for the future?  What about integer schedules?